modules/pam_namespace/pam_namespace.c

```diff
@@ -39,6 +39,95 @@
 #include "pam_namespace.h"
 #include "argv_parse.h"
 
+/* --- evaluting all files in VENDORDIR/security/namespace.d and /etc/security/namespace.d --- */
+static const char *base_name(const char *path)
+{
+    const char *base = strrchr(path, '/');
+    return base ? base+1 : path;
+}
+
+static int
+compare_filename(const void *a, const void *b)
+{
+	return strcmp(base_name(* (const char * const *) a), base_name(* (const char * const *) b));
```

Comment:
This line is too long, please wrap, e.g.
```
        return strcmp(base_name(* (const char * const *) a),
                        base_name(* (const char * const *) b));
```

Reply:
fixed